setup.py

```diff
@@ -1,6 +1,8 @@
 import sys
 import re
+from importlib_metadata import entry_points
```

Comment:
This introduces a new untracked dependency and it is possibly not needed? See [this setup file](https://github.com/mdolab/postprocessing/blob/main/setup.py)

Reply:
Yeah this line does not need to be there, it works without it.